website/source/docs/secrets/ssh/index.html.md

```diff
@@ -305,8 +305,10 @@ The first thing to do is to get Vault to generate the key pair that will be
 used to sign any SSH keys:
 
 ```text
-$ vault write -f ssh/config/ca
-Success! Data written to: ssh/config/ca
+$ vault write -f ssh/config/ca generate_signing_key=true
```

Comment:
`generate_signing_key` will be `true` by default and we don't *need* to specify it.